### PR TITLE
Add emotion

### PR DESCRIPTION
**Pull request by Caruki**
Implement styled components. Turn all css files to styled components (as possible)

### CODE REVIEW

File: src/components/AppHeader.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Logo from './Logo';
+import Title from './Title';
+import styled from '@emotion/styled';
+import Manawallpaper from '../assets/manawallpaper.jpg';
+
+const Header = styled.header`
+  width: 100vw;
```
Comment (marwinburesch) [inline src/components/AppHeader.js]: `AppContainer`, the parent of this already has a `height` of `100vw`

File: src/components/AppHeader.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  font-family: 'MPlantin';
+  font-size: 1rem;
+  flex-grow: 1;
+  margin-top: 7px;
+`;
+
+export default function AppHeader() {
+  return (
+    <Header>
+      <HeaderContainer>
```
Comment (marwinburesch) [inline src/components/AppHeader.js]: probably better to name this `TitleContainer` or something else that'd makes sense. It's not the `HeaderContainer`

File: src/components/AppHeader.js (hunk 1 above)
Comment (Caruki) [reply src/components/AppHeader.js]: changed to width: 100%

File: src/components/AppHeader.js (hunk 2 above)
Comment (Caruki) [reply src/components/AppHeader.js]: changed to `TitleContainer`

**Pull request merged**